keep/api/config.py

```diff
@@ -16,15 +16,22 @@
 def on_starting(server=None):
     """This function is called by the gunicorn server when it starts"""
     logger.info("Keep server starting")
-    
+
     migrate_db()
 
     # Create single tenant if it doesn't exist
     if AUTH_TYPE in [
-        AuthenticationType.SINGLE_TENANT.value,
-        AuthenticationType.NO_AUTH.value,
+        IdentityManagerTypes.DB.value,
```

Comment:
is this right? "DB" is the new single tenant mode?

Reply:
yep - with single_tenant the users are being stored on the db (also authentication) so the "provider" here is db